src/TurbulenceConvection/types.jl

```diff
@@ -457,6 +457,7 @@ struct EDMFModel{
 }
     surface_area::FT
     max_area::FT
+    small_area::FT
```

Comment:
How does this differ from `minimum_area`?

Reply:
only if we have `>=` in the conditions. I changed it to `>` and now we can use only `a_min`